assets/app/views/directives/header/default-header.html

```diff
@@ -0,0 +1,16 @@
+<nav class="navbar navbar-pf-alt" role="navigation">
+  <div class="navbar-header">
+    <div row flex class="navbar-flex-btn toggle-menu">
+      <button type="button" class="navbar-toggle project-action-btn ng-isolate-scope" data-toggle="collapse" data-target=".navbar-collapse-2">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
```

Comment:
Suggest a comment saying why this is here three times. Assuming it's the hamburger menu?